source/features/remove-label-button.tsx

```diff
@@ -0,0 +1,86 @@
+import './remove-label-button.css';
+import React from 'dom-chef';
+import XIcon from 'octicon/x.svg';
+import select from 'select-dom';
+import oneTime from 'onetime';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+import {observe} from 'selector-observer';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+
+import {getRepoURL, getConversationNumber} from '../github-helpers';
+
+const canNotEditLabels = oneTime((): boolean => !select.exists('.sidebar-labels .octicon-gear'));
+
+function updateSidebar() {
+	select('#partial-discussion-sidebar')!.dispatchEvent(new CustomEvent('socket:message', {
+		detail: {
+			name: '',
+			data: {},
+			cached: false
+		}
+	}));
+}
+
+async function removeLabelButtonClickHandler(event: delegate.Event<MouseEvent, HTMLButtonElement>): Promise<void> {
+	event.preventDefault();
+
+	const removeLabelButton = event.delegateTarget;
+
+	removeLabelButton.disabled = true;
+	await api.v3(`repos/${getRepoURL()}/issues/${getConversationNumber()!}/labels/${removeLabelButton.dataset.name!}`, {
+		method: 'DELETE'
+	});
+
+	updateSidebar();
+}
+
+function makeRemoveLabelButton(labelName: string, color: string, backgroundColor: string) {
+	const removeLabelButton = (
+		<button
+			type="button"
+			aria-label="Remove this label"
+			className="btn-link tooltipped tooltipped-nw rgh-remove-label-button"
+			data-name={labelName}
+		>
+			<XIcon/>
+		</button>
+	);
+
+	removeLabelButton.style.setProperty('--rgh-remove-label-button-bg-color', backgroundColor);
+	removeLabelButton.style.setProperty('--rgh-remove-label-button-color', color);
+
+	return removeLabelButton;
+}
+
+async function init(): Promise<void> {
+	await api.expectToken();
+
+	observe('.labels > a', {
+		constructor: HTMLElement,
+		add(label) {
+			// Override !important rule
+			label.style.setProperty('display', 'inline-flex', 'important');
+			label.append(makeRemoveLabelButton(label.dataset.name!, label.style.color, label.style.backgroundColor));
```

Comment:
You can inline it here and use the `style` attribute
```suggestion
			label.append(
				<button
					type="button"
					aria-label="Remove this label"
					className="btn-link tooltipped tooltipped-nw rgh-remove-label-button"
					data-name={labelName},
					style={{
						'--rgh-remove-label-button-bg-color': 'label.style.backgroundColor'
					}}
				>
					<XIcon/>
				</button>
			);
```

Reply:
It doesn't work too. 
`dom-chef` doesn't support css vars in the `style` attribute, because `dom-chef` [uses](https://github.com/vadimdemedes/dom-chef/blob/master/index.ts#L28) `element.style[name as any] = value;` for setting css properties.
![css](https://user-images.githubusercontent.com/36174850/90082999-f86c7600-dd19-11ea-99bc-53f9ccbd8749.gif)

Reply:
Ouch. I opened an issue for that. Let's add a comment:

```js
// TODO: set variable via JSX and inline function in `init` after https://github.com/vadimdemedes/dom-chef/issues/66
```